packages/dd-trace/src/profiling/profiler.js

```diff
@@ -60,6 +62,12 @@ class Profiler extends EventEmitter {
     }
   }
 
+  _nearOOMExport (profileType, encodedProfile) {
+    const start = this._lastStart
+    const end = new Date()
+    this._submit({ [profileType]: encodedProfile }, start, end, SnapshotKinds.OnOutOfMemory)
```

Comment:
This is long enough that I would make the object multiline for readability.

```suggestion
    this._submit({
      [profileType]: encodedProfile
    }, start, end, SnapshotKinds.OnOutOfMemory)
```